### ISSUE TITLE
Warnings in _vertical_rhythm.sass

### ISSUE THREAD

**Issue by codingdesigner** [Compass/compass#464]
I was upgrading an older project that used the Susy version of vertical_rhythm to the lastes Compass 0.11.4 and got a lot of warnings when passing three arguments to mixins like adjust-font-size-to. The logic for triggering the warning appears to be wrong. :

```
// Adjust a block to have a different font size and leading to maintain the rhythm.
// $lines is a number that is how many times the baseline rhythm this
// font size should use up. Does not have to be an integer, but it defaults
// to the smallest integer that is large enough to fit the font.
// Use $from_size to adjust from a non-base font-size.
@mixin adjust-font-size-to($to-size, $lines: lines-for-font-size($to-size), $from-size: $base-font-size) {
  @if $relative-font-sizing and $from-size != $base-font-size {
    @warn "$relative-font-sizing is false but a relative font size was passed to adjust-font-size-to";
  }
  font-size: $font-unit * $to-size / $from-size;
  @include adjust-leading-to($lines, if($relative-font-sizing, $to-size, $base-font-size));
}
```

In my project I called it like:

```
+adjust-font-size-to(14px, 1, 16px)
```

$relative-font-sizing is set to true (by default) and I still got the warnings. 

It looks like you're checking to see if $relative-font-sizing is set to true, but then the warning claims that it's false. 

I wanted to find out what the intention behind that warning was before submitting a pull request.

**Comment by scottdavis** [comment]
this is @ericam 's baby

**Comment by chriseppstein** [comment]
fixed in 0.11.5

**Comment by codingdesigner** [comment]
Glad to see it fixed! Thanks
